src/WormholeJoin.sol

```diff
@@ -57,6 +57,7 @@ contract WormholeJoin {
     DaiJoinLike immutable public daiJoin;
     bytes32     immutable public ilk;
     bytes32     immutable public domain;
+    uint256               public totalDebt;
```

Comment:
nit: I'd place this variable together with the storage ones and leave all the immutable after.